packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -3,6 +3,21 @@ Changelog
 
 Unreleased
 ----------
+### Breaking Changes
```

Comment:
Can we add reasoning for these changes?

Reply:
We removed the breaking change to keep it passive here https://github.com/cerner/terra-framework/pull/899/commits/cd5d87726ffa459e43be295823a935fea67d1f7a?file-filters%5B%5D=.md#diff-cdc6379ea8cdd76265a948289e4bdab4L6